site/content/statistics.md

```diff
@@ -0,0 +1,18 @@
+---
+type: "page"
+title: "Statistics"
+layout: charts
+---
+We measure and record a range of statistics related to ZAP, some of which are made public here.
+
+A blog post will be written soon which explains how this all works.
+
+{{< chart "check-for-updates">}}
+
+The Check for Update requests that ZAP makes are detailed in the FAQ: [What 'calls home' does ZAP make](/faq/what-calls-home-does-zap-make/)
```

Comment:
```suggestion
The Check for Update requests that ZAP makes are detailed in the FAQs: [What 'calls home' does ZAP make?](/faq/what-calls-home-does-zap-make/)
```

Reply:
Hum, not sure about this. In this case I'm refering to a specific "Frequently Asked Question" singular ;)

Reply:
I’d leave it singular and add the question mark

Reply:
I've done that :)